### PR TITLE
Add cron support

### PR DESCRIPTION
**Pull request by kibertoad**
## Checklist

- [X] Apply one of following labels; `major`, `minor`, `patch` or `skip-release`
- [X] I've updated the documentation, or no changes were necessary
- [X] I've updated the tests, or no changes were necessary

### CODE REVIEW

**Review by arthuracs — APPROVED** [review]

File: packages/app/healthcheck-utils/package.json [hunk 1]
```diff
@@ -32,12 +32,14 @@
...
   },
   "peerDependencies": {
+    "@lokalise/background-jobs-common": ">=5.0.0",
     "ioredis": "^5.4.1",
     "prom-client": "^15.1.3",
     "toad-scheduler": "^3.0.1"
   },
   "devDependencies": {
     "@biomejs/biome": "^1.8.3",
+    "@lokalise/background-jobs-common": "^5.2.0",
```
Comment (CarlosGamero) [inline packages/app/healthcheck-utils/package.json:42]: As this is a `major` update, i think we should do this in a different PR to not trigger release

Comment (kibertoad) [reply packages/app/healthcheck-utils/package.json:42]: bah, good catch, this leaked from a different set of changes that I was doing

File: packages/app/background-jobs-common/src/periodic-jobs/AbstractPeriodicJob.ts [hunk 2]
```diff
@@ -49,7 +50,9 @@ export abstract class AbstractPeriodicJob {
       singleConsumerMode: {
         enabled: false,
         exclusiveLockSuffix: options.singleConsumerMode?.exclusiveLockSuffix ?? 'EXCLUSIVE',
-        lockTimeout: options.intervalInMs * 2,
+        lockTimeout: options.schedule.intervalInMs
+          ? options.schedule.intervalInMs * 2
+          : DEFAULT_LOCK_TIMEOUT,
```
Comment (CarlosGamero) [inline packages/app/background-jobs-common/src/periodic-jobs/AbstractPeriodicJob.ts:55]: If we use cron option, `lockTimeout` will always be the default value, is it fine?

Comment (kibertoad) [reply packages/app/background-jobs-common/src/periodic-jobs/AbstractPeriodicJob.ts:55]: not super sure about it, but what would be a better alternative? you can still specify an explicit value if needed, not sure if we need a separate value for specifying a default

Comment (CarlosGamero) [reply packages/app/background-jobs-common/src/periodic-jobs/AbstractPeriodicJob.ts:55]: Mmm, not sure, either, we can go with it for now and if we see it can be a problem then we can add a new value for it

**Review by CarlosGamero — APPROVED** [review]
